ASP.NET Core/Readme.md

```diff
@@ -0,0 +1,54 @@
+# ASP.NET Core
+
+## Installation
+
+Download the example and use Visual Studio 2019 to open the project. 
+
+## Client-side resources and bundling
+
+This project uses [NPM](http://npmjs.com/) and [Gulp.js](https://gulpjs.com/) to install client-side libraries. All required scripts and CSS files will be automatically installed and packaged into the resulted bundle during the project first build.
```

Comment:
Maybe something like this?:
This project uses [NPM](http://npmjs.com/) and [Gulp.js](https://gulpjs.com/) to install client-side libraries. The project restores NPM packages before the first build. Then, gulp bundles required scripts and CSS files into the resulting package during the first and every next build.

The main idea is to include information about the fact that the bundle is recreated every next build.